workers/unity/Assets/Gdk/Core/MutableView/MutableView.cs

```diff
@@ -98,9 +120,9 @@ public void RemoveComponent<T>(long entityId)
             Entity entity;
             if (!TryGetEntity(entityId, out entity))
             {
-                LogDispatcher.HandleLog(LogType.Error, new LogEvent(
-                        "Entity not found when attemting to remove a component from an entity.")
-                    .WithField(IdType.EntityId.ToString(), entityId)
+                LogDispatcher.HandleLog(LogType.Error, new LogEvent(Errors.EntityNotFoundOnRemove)
+                    .WithField(LogDataFieldName.LoggerName, LoggerName)
+                    .WithField(LogDataFieldName.EntityId, entityId)
                     .WithField("OpName", typeof(T).Name));
```

Comment:
move `OpName` also to `LogDatafieldName`

Reply:
at least it seems like it's used often enough

Reply:
Changed :) Also found a few other strings which I replaced.